src/Symfony/Component/Form/ChoiceList/Factory/DefaultChoiceListFactory.php

```diff
@@ -81,6 +81,12 @@ public function createView(ChoiceListInterface $list, $preferredChoices = null,
                     $otherViews
                 );
             }
+
+            if ($groupByOrder && is_array($groupByOrder)) {
+                uksort($otherViews, function ($key1, $key2) use ($groupByOrder) {
```

Comment:
@colinodell , thanks 
Can you give me an example of callable groupByOrder you need ?
Which parameters you need in the function ?